src/behaviors/clap/clap.js

```diff
@@ -1,28 +1,28 @@
 import Behavior from '../behavior.js';
 
-class RollTheDice extends Behavior {
+class ClaporRave extends Behavior {
   constructor(settings) {
-    settings.name = 'Clap Hype';
-    settings.description = `'cause 👏 sometimes 👏 you 👏 need 👏 emphasis!`;
+    settings.name = 'ClappingRave';
+    settings.description = `'cause sometimes you need emphasis!`;
     super(settings);
 
-    this.commands.push({
-      tag: 'clap',
-      description: `I'll hype your message for you 👏👏👏`
-    });
+    ['clap', 'rubyrave', 'wut'].forEach(emoji => this.commands.push({
+      tag: emoji,
+      description: `I'll hype your message, !Clap or !Rave`
+    }));
   }
 
   execute(command, message, channel, data) {
-    const parsedMessage = this.parseMessage(message);
+    const parsedMessage = this.parseMessage(message, command);
 
-    this.bot.postMessage(channel, `${parsedMessage.join(' :clap: ')} :clap:`, {
+    this.bot.postMessage(channel, `${parsedMessage.join(' :' + command + ': ')} :${command}:`, {
```

Comment:
Neat thing about template literals is that you can nest them!

```js
`${parsedMessage.join(`:${command}: `)} :${command}:`
```

Reply:
_You can?!_ I was scared ;-;

Reply:
Woohoo naive guessing worked!